### ISSUE TITLE
properties in the input msg are not returned in msg2

### ISSUE THREAD

**Issue by juggledad** [bartbutenaers/node-red-contrib-wav-headers#3]
The input msg properties - other than msg.payload - are not returned in output 2. i.e. If you pass in a msg with msg.filename into the node and the node is set to 'Get WAV headers', in output 1 you will see msg.filename while in output 2 it will not be there. 

Looking at the code in wav_headers.js line 62 is
`            var msg2 = {};`
so I changed it to 
`            var msg2 = RED.util.cloneMessage(msg);
`
and that seems to work. Do you see any reason that it will cause a problem? If Not, I'll do a PR if you want.

The reason I'm looking for the other properties in msg2 is I want to have the filename and the time of a wav to pass on to another flow that plays the sound and controls a servo for the jaws of a skeleton's head

**Comment by bartbutenaers** [comment]
That is something that I often forget...  
Just use the input msg, without cloning!  Because with audio, LOTS of data is involved.  When you clone each sample , that would cause a lot of unnecessary overhead.

Node-RED will do the cloning: 
+ On the first output wire, the original output msg will be passed.
+ On all other output wires, a clone of the output msg will be passed.

So if you make a long chain of nodes (i.e. 1 wire at each output), then we shouldn't have any cloning at all.  Which is what we want, especially for large messages (e.g. audio and video).

**Comment by juggledad** [comment]
Why not just have one output and put the WAV header information in another part of the msg like msg.wav_header?

**Comment by bartbutenaers** [comment]
I have no idea why have implemted it like that...
Damn that will require a breaking change.

**Comment by juggledad** [comment]
If you add the wav header information to the msg coming out of output 1, this wouldn't this just be a new feature and not a breaking change?

**Comment by bartbutenaers** [comment]
Of course, but then then the second output becomes completely irrelevant.  So need to remove that.  Anyway, I will make it a major version so people so that there might be breaking changes, and put it on the readme page for those who don't know the meaning of major versions...
